sites/partners/public/images/icons.svg

```diff
@@ -236,9 +236,7 @@
  <path fill-rule="evenodd" clip-rule="evenodd" d="M.205 3.483L3.35.105c.136-.14.359-.14.496 0l.496.508a.366.366 0 0 1 0 .508l-2.4 2.616a.366.366 0 0 0 0 .508l2.4 2.634a.366.366 0 0 1 0 .508l-.496.508a.345.345 0 0 1-.496 0L.205 4.499a.73.73 0 0 1 0-1.016z" fill="#0077DA"/>
  </symbol>
 <symbol id="i-spinner" viewBox="0 0 100 100">
-<circle cx="50" cy="50" fill="none" stroke="currentColor" stroke-width="9" r="32" stroke-dasharray="150.79644737231007 52.26548245743669">
-  <animateTransform attributeName="transform" type="rotate" repeatCount="indefinite" dur="1s" values="0 50 50;360 50 50" keyTimes="0;1"></animateTransform>
-</circle>
+<circle cx="50" cy="50" fill="none" stroke="currentColor" stroke-width="9" r="32" stroke-dasharray="150.79644737231007 52.26548245743669"></circle>
```

Comment:
Probably conflict with: https://github.com/bloom-housing/bloom/pull/1230

Reply:
Good point, I'll suggest a change to 1230 to incorporate this.